### DISCUSSION TITLE
Error while using UERANSIM || UE does not have an emergency

### DISCUSSION THREAD

**Discussion by Deblina96** [OAI/OpenAPI-Specification#3498] [Q&A]
I am trying to build my UE and ran using UERANSIM. I am following the below document:
https://gitlab.eurecom.fr/oai/cn5g/oai-cn5g-fed/-/blob/master/docs/DEPLOY_SA5G_WITH_UERANSIM.md

But in the document is mentioned that, ##### IMPORTANT: Add following parameters in oai-amf service of docker-compose, before deploying core network.

```shell
            - INT_ALGO_LIST=["NIA1" , "NIA2"]
            - CIPH_ALGO_LIST=["NEA1" , "NEA2"]
```
My question is it, in which section of aoi-amf should I add it ?

I have tried to add it like in below mention SS, but did not work:
![image](https://github.com/OAI/OpenAPI-Specification/assets/116576465/5d32182e-3537-4f3e-815f-9164b7647107)

Error:
![image](https://github.com/OAI/OpenAPI-Specification/assets/116576465/82f7c34a-ca35-49d1-9e07-e9e95e52910d)

Please suggest on this.

**Comment by lornajane** [comment]
This repository is for the discussion and development of the OpenAPI standard only, please direct your questions to the project whose package you are using as I believe they will be better placed to assist.
